commands/setup.go

```diff
@@ -139,11 +139,15 @@ func jaegerConfigFromCliContext(cctx *cli.Context) (*jaegerConfig, error) {
 }
 
 func setupLogging(cctx *cli.Context) error {
-	ll := cctx.String("log-level")
-	if err := logging.SetLogLevel("*", ll); err != nil {
+	//ll := cctx.String("log-level")
+	if err := logging.SetLogLevel("*", "error"); err != nil {
 		return xerrors.Errorf("set log level: %w", err)
 	}
 
+	if err := logging.SetLogLevelRegex("visor/*", "debug"); err != nil {
+		panic(err)
```

Comment:
Panic doesn't seem right here. Why not return an error which will be reported to user running command.